<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskListView</class>
 <widget class="QWidget" name="TaskListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 65, 66);
color: rgb(208, 208, 208);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QToolButton" name="user">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>用户名</string>
           </property>
           <property name="icon">
            <iconset resource="../resource.qrc">
             <normaloff>:/icons/user.png</normaloff>:/icons/user.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>78</width>
             <height>78</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="tasktypeListWidget">
        <property name="styleSheet">
         <string notr="true">QListWidget {
	outline: 0;
	border: none;
 	background-color: rgb(64, 65, 66);
}

QListWidget::item {
	color: white;
	border: none;
}

QListWidget::item:hover {
	background-color: rgb(94, 95, 96);
}

QListWidget::item:selected {
	background-color: rgb(33, 34, 34);
}</string>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="selectionRectVisible">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>未完成的任务</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icons/doing.png</normaloff>:/icons/doing.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>已完成的任务</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icons/done.png</normaloff>:/icons/done.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>可领取的任务</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icons/unassigned.png</normaloff>:/icons/unassigned.png</iconset>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 47, 48);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:disabled
{
	color: rgb(99, 100, 101);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="cmbAssayType">
           <item>
            <property name="text">
             <string>所有类型</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>形态</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>骨髓</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>外周血</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editCaseNumber">
           <property name="placeholderText">
            <string>病历号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnQuery">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUntake">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>退领</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnTake">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>领取</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>193</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="cmbDeviceId">
           <item>
            <property name="text">
             <string>设备1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>设备2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>设备3</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRefresh">
           <property name="text">
            <string>刷新</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="taskTableView">
        <property name="styleSheet">
         <string notr="true">QTableView {

}

QTableView QHeaderView::section{
	color: white;
	background-color: rgb(76, 77, 77);
}

QTableView QTableCornerButton::section {
	color: white;
	background-color: rgb(76, 77, 77);
}
</string>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
